pkgs/development/libraries/libgpiod/default.nix

```diff
@@ -11,6 +11,12 @@ stdenv.mkDerivation rec {
     sha256 = "14iv8iwyvfznyifhkqz1bjh24dvb03gmpjn8v3fs6h734l93vvw5";
   };
 
+  patches = [
+    # cross compiling fix
+    # https://github.com/brgl/libgpiod/pull/45
+    ./0001-Drop-AC_FUNC_MALLOC-and-_REALLOC-and-check-for-them-.patch
```

Comment:
Would this fix the problem maybe as well? https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/filesystems/nilfs-utils/default.nix#L40

Reply:
Since the patched is merged though, this should be fine.